### PR TITLE
CIFS Backup Store Support

### PR DESCRIPTION
**Pull request by derekbit**
https://github.com/longhorn/longhorn/issues/3599

### CODE REVIEW

**Comment by derekbit** [comment]
Dependencies are updated.

File: pkg/util/util.go [hunk 1]
```diff
@@ -265,33 +264,12 @@ func CheckBackupType(backupTarget string) (string, error) {
 }
 
 func GetBackupCredential(backupURL string) (map[string]string, error) {
```
Comment (innobead) [inline pkg/util/util.go:266]: Suggest moving `GetBackupCredential` to util/credential.go as well.

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, just one minor suggestion.

File: pkg/util/util.go (hunk 1 above)
Comment (derekbit) [reply pkg/util/util.go]: Addressed. Thank you.

**Review by innobead — APPROVED** [review]

**Review by shuo-wu — APPROVED** [review]

**Pull request merged**